crates/bin/node/Cargo.toml

```diff
@@ -26,7 +26,7 @@ clap = { workspace = true, features = ["derive"] }
 tonic = { workspace = true }
 tonic-health = { workspace = true }
 prost = { workspace = true }
-nuts = { workspace = true }
+nuts = { workspace = true, features = ["nut19"] }
```

Comment:
do we actually use this? If not just remove it. And the whole file with it maybe.
Or move the caching logic there